charts/rancher-monitoring/v0.0.7/charts/prometheus/templates/service.yaml

```diff
@@ -18,4 +18,8 @@ spec:
   ports:
     - name: http
       port: 80
-      targetPort: http
+      targetPort: 8080
+    - name: grpc
+      port: 10901
+      protocol: TCP
+      targetPort: 10901
```

Comment:
`10901` is not a container port of the Prometheus, can u please help to clarify what is this for?

Reply:
Thanos sidecar will be added to the prometheus pod, and this is the port that thanos sidecar expose.

Reply:
Thanos sidecar will be added to the prometheus pod, and this is the port that thanos sidecar expose.